modules/admin/proc-install-operator.adoc

```diff
@@ -19,23 +23,32 @@ Containers are available for the following CPU architectures:
 * You are logged in as an administrator on the {ocp-short} web console.
 * You have configured the appropriate roles and permissions within your project to create an application. For more information, see the link:https://docs.openshift.com/container-platform/{ocp-version}/applications/index.html[Red Hat OpenShift documentation on Building applications].
 
-.Procedure
+[NOTE]
+====
+For enhanced security, deploy the {product} Operator in a dedicated default namespace such as `rhdh-operator`. The cluster administrator can restrict other users' access to the operator resources through role bindings or cluster role bindings. You can choose to deploy the operator in the `openshift-operators` namespace instead, however, the {product} operator shares the namespace with other operators, and therefore any users who can create workloads in that namespace can get their privileges escalated from all Operators' service accounts.
+====
 
-. In the *Administrator* perspective of the {ocp-short} web console, navigate to *Operators* > *OperatorHub*.
+.Procedure
 
+. In the *Administrator* perspective of the {ocp-short} web console, navigate to *Operators* -> *OperatorHub*.
 . In the *Filter by keyword* box, enter {product-short} and click the *{product}* Operator card.
-
 . On the *{product} Operator* page, click *Install*.
 
-. On the *Install Operator* page, click *Install*.
+. On the *Install Operator* page, use the *Update channel* drop-down menu to choose an update channel:
+** The *fast* channel provides minor version (y-stream) updates, for example, updating from version 1.0 to 1.1.
```

Comment:
```suggestion
** The *fast* channel provides minor version (y-stream) updates, for example, updating from version 1.0 to 1.1 to 1.2 and beyond.
```

Reply:
Updated and added a note per Slack thread with @rm3l